.gitignore

```diff
@@ -21,11 +21,9 @@ vendor
 /conf.log
 /conf.pri
 /Makefile
-/src/cpp/Makefile
-/src/cpp/cpp/Makefile
-/src/cpp/tests/Makefile
 /src/runner/certs/*.crt
 /src/runner/certs/*.key
+/postbuild/conf.pri
```

Comment:
if we want all conf.pri files to be ignore we can add `conf.pri` once in .gitignore